src/uart.rs

```diff
@@ -0,0 +1,331 @@
+//! Access to [RIOT's UART](https://doc.riot-os.org/group__drivers__periph__uart.html)
+//!
+//! Author: Kilian Barning <barning@uni-bremen.de>
```

Comment:
We don't have a consistent policy in Rust code for RIOT as to how to annotate authorship per module. I'd leave this in for now, especially until we do have something more generic, but as this is not a copyright note, it may be removed in a later version – realistically, I expect it to be moved into more meta data at some point.